guides/common/modules/proc_creating-a-shellhook-to-print-arguments.adoc

```diff
@@ -3,12 +3,12 @@
 
 Create a simple shellhook script that prints `Hello World!` when you run a remote execution job.
 
-.Prerequisites
-* You have the webhooks and shellhooks plug-ins installed.
+.Prerequisite
```

Comment:
We have also changed this to plural globally. The diff should go away when you rebase.

Reply:
Looks like it was missed. Hope it's OK if I fix it in this PR.